### ISSUE TITLE
Undefined variable: g:ale_emit_conflict_warnings

### ISSUE THREAD

**Issue by levylll** [w0rp/ale#426]
when I open a file, I get this error
what should I do ?~
```
Error detected while processing /Users/levy/.vim/bundle/ale/after/plugin/ale.vim:
line    7:
E121: Undefined variable: g:ale_emit_conflict_warnings
E15: Invalid expression: !g:ale_emit_conflict_warnings
Press ENTER or type command to continue
```

**Comment by janclarin** [comment]
I'm getting this as well. If `g:ale_emit_conflict_warnings` isn't specified in .vimrc, this error appears. A temporary fix would be to add `g:ale_emit_conflict_warnings=0` to your .vimrc.

**Comment by janclarin** [comment]
I've found that this issues occurs on older versions of vim (7.4), but not on the latest ones (8.0). Can you check your vim version?

**Comment by levylll** [comment]
@janclarin I use vim  8.0.512...

**Comment by janclarin** [comment]
Submitted a PR to fix this: #431

**Comment by janclarin** [comment]
**One way to reproduce the bug**
Run an older version of vim (before 8.0) so that [this block](https://github.com/w0rp/ale/blob/master/plugin/ale.vim#L24) is entered and [calls finish](https://github.com/w0rp/ale/blob/master/plugin/ale.vim#L32) before running after/plugin/ale.vim.

If [`finish`](https://github.com/w0rp/ale/blob/master/plugin/ale.vim#L32) is called then `g:ale_emit_conflict_warnings` is never set to any value since it is only initialized [after the finish call](https://github.com/w0rp/ale/blob/master/plugin/ale.vim#L48). So, when [line 7](https://github.com/w0rp/ale/blob/master/after/plugin/ale.vim#L7) is called in after/plugin/ale.vim the following error is produced

```
Error detected while processing /Users/jan/.vim/plugged/ale/after/plugin/ale.vim:
line    7:
E121: Undefined variable: g:ale_emit_conflict_warnings
E15: Invalid expression: !g:ale_emit_conflict_warnings
Press ENTER or type command to continue
```
